tests/decodeinputavformat.cpp

```diff
@@ -124,7 +126,11 @@ bool DecodeInputAvFormat::getNextDecodeUnit(VideoDecodeBuffer &inputBuffer)
     int ret;
     while (1) {
         //free old packet
+#if (__LIBAVCODEC_NEW__)
```

Comment:
Please check which version will rename av_free_packet, and use following define at start

```
#if LIBAVCODEC_VERSION_INT < AV_VERSION_INT(xxxx)		
#define av_packet_unref av_free_packet
 #endif
```